cognite/neat/rules/exporter/rules2ontology.py

```diff
@@ -18,6 +20,39 @@
     from typing_extensions import Self
 
 
+class OWLExporter(BaseExporter):
+    def __init__(self, rules: Rules, filepath: Path):
+        self.filepath = filepath
+        self.rules = rules
+        self.data = Ontology.from_rules(self.rules).as_owl()
+        super().__init__(self.rules, self.filepath)
+
+    def export(self):
+        self.data.serialize(destination=self.filepath)
```

Comment:
Same here, lets keep the init for setting up, and the `.export` for the logic
```suggestion
        super().__init__(self.rules, self.filepath)

    def export(self):
        data = Ontology.from_rules(self.rules).as_owl()
        data.serialize(destination=self.filepath)
```